api/controllers/game/ready.js

```diff
@@ -1,3 +1,5 @@
+const gameService = require('../../services/gameService');
+
```

Comment:
Sails exposes the services as globals by default so we don't need an explicit import here. Granted in general I would prefer explicit imports, but keep it consistent with the rest of the app for now
```suggestion
```

Reply:
I'd actually prefer keeping the import since it allows me to click into the function to see its definition in VSCode. It comes in handy sometimes and provides better devx.

Reply:
I hear you and I think it's a pain  point worth discussing further. I don't particularly like the idea of it being imported explicitly some of the time and not others, though because an inconsistent style is harder to follow. So we could potentially roll it out as an app-wide change but I didn't want to increase the scope of this pr

Reply:
Agreed it's better to make it an app-wide change. I'll keep them for now for debugging purposes and remove them before release